app/src/main/java/org/stepic/droid/ui/fragments/FindCoursesFragment.kt

```diff
@@ -0,0 +1,83 @@
+package org.stepic.droid.ui.fragments
+
+import android.os.Bundle
+import android.support.annotation.StringRes
+import android.view.*
+import io.reactivex.disposables.CompositeDisposable
+import org.stepic.droid.R
+import org.stepic.droid.analytic.Analytic
+import org.stepic.droid.base.CoursesDatabaseFragmentBase
+import org.stepic.droid.storage.operations.Table
+import org.stepic.droid.ui.custom.AutoCompleteSearchView
+import org.stepic.droid.ui.util.SearchViewHelper
+import org.stepic.droid.ui.util.initCenteredToolbar
+
+open class FindCoursesFragment: CoursesDatabaseFragmentBase() {
```

Comment:
this class will be removed in parallel PR, so I won't review this, but I will resolve merge conflicts, when this branch will be merged to explore pr